lib/index.js

```diff
@@ -1,3 +1,5 @@
+import { Element as PolymerElement } from '../../@polymer/polymer/polymer-element';
```

Comment:
This is wrong. `lib/index.js` is generated from `npm run build`.
Also why are you including this here when we don't use PolymerElement anywhere?

Reply:
I should maybe document this on the README actually of how to build the repo.

Reply:
didn't know that... ok maybe it's better if I leave the repo adaptation to v3 to you, the main thing that is required to make polymer-redux working in a v3 project was that 

```javascript
import {get} from '../../@polymer/polymer/lib/utils/path';
```

thing, so in the meanwhile I can work with my fork.
Sorry for the inconvenience

Reply:
No, don't be sorry. I'm snowed under at work, this is great I really appreciated it 👍 

Just run `npm run build` then commit the changes and you're done 🎉